index.html

```diff
@@ -358,66 +316,149 @@
       </section>
       <section>
         <h2>
-          <code>CanvasImageSource</code> typedef
+          <code>DepthData</code> interface
```

Comment:
Would you consider to use DepthMap name? As depthmap (http://en.wikipedia.org/wiki/Depth_map) is a quite common name used in 3D camera space, e.g. Google Tango Project (https://developers.google.com/depthmap-metadata/) and Intel RealSense SDK are all using it in developer manual.